tpm2/constants.go

```diff
@@ -128,10 +129,11 @@ const (
 )
 
 const (
-	tagNull       tpmutil.Tag = 0x8000
-	tagNoSessions tpmutil.Tag = 0x8001
-	tagSessions   tpmutil.Tag = 0x8002
-	tagHashcheck  tpmutil.Tag = 0x8024
+	tagNull          tpmutil.Tag = 0x8000
+	tagNoSessions    tpmutil.Tag = 0x8001
+	tagSessions      tpmutil.Tag = 0x8002
+	tagAttestCertify tpmutil.Tag = 0x8017
+	tagHashcheck     tpmutil.Tag = 0x8024
```

Comment:
nit: I see that I missed this in previous reviews: shouldn't this be `tagHashCheck`? (capitalized `C`)

Reply:
Done